app/assets/javascripts/cascade/new-home.js

```diff
@@ -0,0 +1,1195 @@
+
+				// Need to wait until the transition class is in the DOM
+				setTimeout(function() {
+					$new_content.css({
+						"opacity":1
+					});
+
+					$old_content.css({
+						"opacity":0
+					});
+				},250, $new_content, $old_content);
+				
+				// Wait until the transition is complete
+				setTimeout(function() {
+					// Remove the old content from the DOM
+					$old_content.remove();
+
+				},cu_hero_area.contentTransitionSpeed, $old_content);
+
+
+			} else {
+				$("#hero").before(raw_html).remove();
+			}
+
+			cu_hero_area.setupContent($new_content);
+
+			if (direction == 'newer') {
+				cu_hero_area.queueTitleAndExcerptEntrance(cu_hero_area.contentTransitionSpeed / 2.5);
+			} else {
+				cu_hero_area.queueTitleAndExcerptEntrance(cu_hero_area.contentTransitionSpeed / 3.3);
+			}
+
+
+			var timeToHideNav = (cu_window_manager.useTransitions) ? cu_hero_area.contentTransitionSpeed : 10;
+
+			// Tasks after the switch is complete
+			setTimeout(function() {
+				var current_num = cu_hero_area.currentCampaign;
+				var max = cu_hero_area.pastCampaigns.length -1; // zero based
+				var min = 0;
+
+				$("#mastheadNavigation").fadeIn();
+				if (current_num == min) $("#showNewerContent").addClass('disabled');
+				if (current_num == max) $("#showOlderContent").addClass('disabled');
+
+				// Show the buttons if we aren't at an end of the list
+				if (current_num != min) $("#showNewerContent").removeClass('disabled');
+				if (current_num != max) $("#showOlderContent").removeClass('disabled');
+
+				cu_hero_area.isChanging = false;
+
+			}, timeToHideNav);
+			
+
+		}
+
+	} // end cu_hero_area
+
+
+	var cu_stories_area = {
+
+		initialize : function() {
+			// Share expansion
+			$(".stories").delegate('.meta .shares','click',function(e) {
+				$(this).parent(".meta").addClass("share_action");
+				e.preventDefault();
+				return false;
+			})
+
+			$(".meta .shares").lazybind("mouseenter", function(e) {
+				
+				$(".stories .meta").removeClass("share_action"); // clean up open items
+
+				$(e.currentTarget).parent(".meta").addClass("share_action");
+			}, 400, "mouseleave");
+
+			$(".meta").lazybind("mouseleave", function(e) {
+				// $(e.currentTarget).removeClass("share_action");
+				$(".stories .meta").removeClass("share_action");
+			}, 600, "mouseenter");
+
+
+			// Set up the social media sharing function
+			$(".stories").delegate('.meta a','click', this.processShareClick);
+
+
+		},
+
+		processShareClick : function(e) {
+			if (e.target.href !== undefined) {
+				var url = e.target.href;
+				var service = $(e.target).attr('service');
+			} else {
+				var url = $(e.target).parents("a").attr("href");
+				var service = $(e.target).parents("a").attr('service');
+			}
+
+			// if (service == 'Comments') {
+
+			// 	_gaq.push(['_trackEvent', 'Social Engagement', 'Comment Jump Button', service]);
+
+			// 	setTimeout(function() {
+			// 		document.location.href = url;
+			// 	}, 200);
+
+			// 	return false;
+			// }

```

Comment:
This was probably already there, but we should remove dead code. If there is some compelling reason for keeping it temporarily, include a comment.